.aios-core/core/config/env-interpolator.js

```
@@ -0,0 +1,122 @@
+
+  function walk(obj, path) {
+    if (typeof obj === 'string' && ENV_VAR_PATTERN.test(obj)) {
+      // Reset regex lastIndex after test()
+      ENV_VAR_PATTERN.lastIndex = 0;
+      findings.push(`${sourceFile}: ${path} contains env variable pattern: ${obj}`);
+    } else if (isPlainObject(obj)) {
+      for (const [key, value] of Object.entries(obj)) {
+        walk(value, path ? `${path}.${key}` : key);
+      }
+    } else if (Array.isArray(obj)) {
+      obj.forEach((item, i) => walk(item, `${path}[${i}]`));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)